WBooks/AddNew/AddNewController.swift

```diff
@@ -7,23 +7,165 @@
 //
 
 import UIKit
+import ReactiveSwift
+import ReactiveCocoa
+import Result
+import WolmoCore
 
 class AddNewController: UIViewController {
-
+    
+    private let formFilled = MutableProperty(false)
+    
+    private var loading = false
+    
+    private lazy var _view: AddNewView = AddNewView.loadFromNib()!
+    
+    private var _viewModel: AddNewViewModel = AddNewViewModel()
+    
+    override func loadView() {
+        view = _view
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        _view.setView()
+        setBindings()
+        setFormValidation()
+        setSubmitButton()
+        setImagePicker()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        configureNavBar()
```

Comment:
Why you need to do this each time the view appears and not in viewDidLoad?

Reply:
This is because I need to change the title and buttons of the nav bar for each tab bar view. That nav bar belongs (I understand it does) to the tab controller, so if I include that code on view did load it configures the nav bar only when the view is loaded to memory, and thus the nav bar items don't change in subsequent view changes from the tab bar.

Reply:
I think each of the should be in a different navigation controller